.travis.yml

```diff
@@ -92,6 +93,20 @@ matrix:
     - os: windows
 
 script:
+  - |
+    if [ "$TRAVIS_BRANCH" == "auto" ] || [ "$TRAVIS_BRANCH" == "try" ]; then
+      pr=$(echo $TRAVIS_COMMIT_MESSAGE | grep -o "#[0-9]*" | head -1 | sed 's/^#//g')
+      output=$(curl -H "Authorization: token $GITHUB_API_TOKEN" -s "https://api.github.com/repos/rust-lang/rust-clippy/pulls/$pr" | \
+        python -c "import sys, json; print(json.load(sys.stdin)['body'])" | \
+        grep "changelog: " | \
```

Comment:
should this be tightened to be "^changelog: "? or would that be too restrictive?

Reply:
Yeah, I wondered the same. E.g. #4210 would fail, because the changelog is italic.

Reply:
Done 👍